tests/lava/proc/conv/test_models.py

```diff
@@ -62,6 +62,7 @@ def run_spk(self):
             data = self.data[..., (self.current_ts - 1) % buffer]
             print(f'Sending data ={data[data!=0]}')
         self.s_out.send(self.data[..., (self.current_ts - 1) % buffer])
+        self.s_out.flush()
```

Comment:
This command does not do anything right now. And once it does, you will have to modify the Process a bit to not send dummy data in the end.